app/controllers/api/v4/converts_controller.rb

```diff
@@ -7,6 +7,13 @@ def create
 
     @run = Run.new(run_file: run_file, user: nil)
     @run.parse(fast: ["on", "1"].include?(params[:historic]) ? false : true, convert: true)
```

Comment:
I know this is from before but I'm just noticing this -- `a ? false : true` can be simplified to `!a`. Also I'd prefer supporting only `historic=1` and not `historic=on` in order to simplify the code and have consistency with the other endpoint with `historic`.